src/components/WwwFrame/TheHeader.vue

```diff
@@ -502,35 +521,18 @@ export default {
 			}
 			return `/ui-login?doneUrl=${encodeURIComponent(this.$route.fullPath)}`;
 		},
-		headerTheme() {
-			return this.theme ? this.theme : lightHeader;
-		},
-		cssVars() {
-			if (this.headerTheme) {
-				return {
-					'--kv-header-background-color': this.headerTheme.backgroundColor || '',
-					'--kv-header-text-color': this.headerTheme.textColor || '',
-					'--kv-header-logo-color': this.headerTheme.logoColor || '',
-					'--kv-header-link-color': this.headerTheme.linkColor || '',
-					'--kv-header-link-hover-color': this.headerTheme.linkHoverColor || '',
-					'--kv-header-separator-color': this.headerTheme.separatorColor || '',
-					'--kv-header-shadow': this.headerTheme.shadow || '',
-				};
-			}
-			return {};
-		}
 	},
 	apollo: {
 		query: headerQuery,
 		preFetch: true,
 		result({ data }) {
-			this.isVisitor = !_get(data, 'my.userAccount.id');
-			this.isBorrower = _get(data, 'my.isBorrower');
-			this.loanId = _get(data, 'my.mostRecentBorrowedLoan.id');
-			this.trusteeId = _get(data, 'my.trustee.id');
-			this.basketCount = _get(data, 'shop.nonTrivialItemCount');
-			this.balance = Math.floor(_get(data, 'my.userAccount.balance'));
-			this.profilePic = _get(data, 'my.lender.image.url');
+			this.isVisitor = !data?.my?.userAccount?.id ?? false;
```

Comment:
I'm a bit confused by this line because it's not clear to me if the `!` will be applied to all of `data?.my?.userAccount?.id ?? false`, or just the `data?.my?.userAccount?.id` part. I think this could also be written as `this.isVisitor = !data?.my?.userAccount?.id;`, which would avoid that question altogether.